app/src/main/java/org/xbmc/kore/utils/Utils.java

```diff
@@ -250,4 +253,28 @@ public void onError(int errorCode, String description) {
             }
         }, callbackHandler);
     }
+
+    public static void setLocale(Context context, String localeName) {
+        Locale locale = getLocale(localeName);
+
+        Locale.setDefault(locale);
+
+        Resources resources = context.getResources();
+
+        Configuration configuration = resources.getConfiguration();
+        configuration.locale = locale;
+
+        resources.updateConfiguration(configuration, resources.getDisplayMetrics());
+    }
+
+    public static Locale getLocale(String localeName) {
+        Locale locale;
+        String[] languageAndRegion = localeName.split("_", 2);
+        if (languageAndRegion.length > 1) {
+            locale = new Locale(languageAndRegion[0], languageAndRegion[1]);
+        } else {
+            locale = new Locale(localeName);
+        }
```

Comment:
There's a third constructor with a third parameter - variation. Not sure if it's widely used though.

Reply:
I have no idea where variant is used for and where to get it. So I suggest we leave it out.